Assets/Plugins/Source/Core/Utility/FileUtility.cs

```diff
@@ -32,27 +32,58 @@ namespace PlayEveryWare.EpicOnlineServices.Utility
     /// </summary>
     public static class FileUtility
     {
+        // Note: "\r\n" (Windows), "\n" (Unix/Linux), "\r" (older Macs)
+        private static readonly string[] LineEndings = new[] { "\r\n", "\n", "\r" };
```

Comment:
This exists to emulate the same functionality provided by C#'s `File.ReadAllLines` functions. Since Android (and perhaps later other platforms) may need special implementations surrounding IO, the function is built off of the existing `ReadAllText` function within `FileUtility`. The way that C# implements `File.ReadAllLines` is that it breaks lines on any and all of these characters (even if the file contents have mixed line endings).